### DISCUSSION TITLE
Wie schützen wir unsere Daten aus der eID?

### DISCUSSION THREAD

**Discussion by pascalgottret** [e-id-admin/governance-sounding-board#39] [General]
Die Piratenpartei, die den jetzigen Vorschlag zur eID ablehnt, bringt es auf den Punkt: „Ein schleichend eingeführtes „Zeig mir en Uswiis“ für alltägliche Dinge muss ausgeschlossen werden.“

Seit längerer Zeit bin ich erfolglos auf der Suche nach der Antwort, wie wir verhindern können, dass das eID Ecosystem mit den drei Ambitionsniveaus ausgenutzt wird, um Daten zu sammeln und korrelieren zu können. Mit einem etablierten Ecosystem ist es für Verifier sehr einfach möglich Daten zu erzwingen (da sie sonst ihren Service ggf. verwehren), zu speichern und zu verkaufen. 

(Beispiel heute: für ein Online Ticket von der SBB muss man den Vor- und Nachnamen hinterlegen. Wozu ist das notwendig? Meines Erachtens gibt es keinen Grund dafür.)

Es gibt Ansätze zur Unterbindung:
- Verifier müssen in der Trusted Registry registriert sein, für die ein klares Governance Framework ausgearbeitet wird. Eher bürokratisch und utopisch, da Skalierbarkeit verloren geht.
- Eine unabhängige Meldestelle, um Verstösse melden zu können. Doch wer bestimmt, was ein Verstoss ist?
- Proof requests für den eindeutigen Beweis, dass ein Verifier bestimmte Daten angefordert hat.
- ZKP bei gesetzlich nicht geregelten Sachverhalten.

Reicht das allerdings aus? 
Reicht Art. 10 des BGEID in dieser Form so aus oder wie sinnvoll ist der Vorschlag der Partei: "Deshalb fordern die Piraten explizit die Präzisierung in Art. 10, dass eine E-ID nur verlangt werden darf, wenn der Gesetzgeber eine Ausweispflicht vorsieht (z.B. Eröffnung eines Bankkontos) und nur diese Daten übertragen werden dürfen, die für diesen Prozess auch wirklich benötigt werden."?

Dürfte ich das Projektteam darum bitten diesen immens wichtigen Punkt nicht zu vernachlässigen? Gibt es bereits antworten? Es droht die Gefahr, dass wir hier auf ein berechtigtes Referendum stossen, wenn dieser Punkt nicht geklärt ist. 
Er macht mir echte Bauchschmerzen.

Mir ist bewusst, dass im Forum hier (https://github.com/e-id-admin/governance-sounding-board/discussions/22) das schon angesprochen wurde. Aber leider geht die Diskussion nicht weiter. Ich versuche nochmal hiermit auf die Dringlichkeit dieses Problems zu hinzuweisen.

**Comment by grt-dti** [comment]
Du sprichst mir aus der Seele.... genau dieses Szenario ist der (potentielle) Todesstoss für die ganze SSI-Idee, weil genau die ZKP-Fähigkeit der einzig reale (und unwiderlegbare, nicht weg-diskutierbare) Wert gegenüber PKI- und IdP-basierten Lösungen ist. Wenn diese lokal durchsetzbare Privacy in Realität durch genau diese Marktmachtmissbräuche geschliffen wird, brauchen wir auch den ganzen Aufwand mit der SSI nicht mehr - dann haben wir einfachere und verlässlichere Lösungen.

Wenn man die E-ID schon als einen Ausweis konzipiert (und verkauft), dann sollten auch die Regeln wie bei einem Ausweis gelten - folglich muss Artikel 10 verschärft und Ausweisgesetz angepasst werden.....

**Comment by sspreitzer** [comment]
> Beispiel heute: für ein Online Ticket von der SBB muss man den Vor- und Nachnamen hinterlegen. Wozu ist das notwendig? Meines Erachtens gibt es keinen Grund dafür.

Dies liegt vermutlich in der Art und Weise wie Billette der SBB physikalisch, wie auch digital produziert werden. So gibt es personifizierte digitale und nicht-personifizierte physikalische Billette.

Digitale nicht-personifizierte Billette wären in der Praxis beliebig und rasch zu vervielfältigen und könnten somit leicht missbraucht werden. Es muss daher für die Kontrolle personifizierter Billette möglich sein, die Person zu identifizieren und dies geht leider nur mit einer auf die Person zeigende Information (Heute oft Vorname, Nachname & Geburtsdatum).

Nun lässt sich argumentieren ob diese Information direkt sein muss (Vorname, Nachname & Geburtsdatum) oder indirekt (z.B. Kundennummer oder Einmal-hash). Dies birgt aber auch Nachteile mit sich, wie z.B. dass diese Billette nichtmehr ohne Zugang zu Online Systemen verifizieren lassen (z.B. in Tunneln oder Strecken ohne Netzabdeckung).

Kurzum, die SBB verwendet persönliche Informationen auf den Billetten nicht vorsätzlich für ein niederes Interesse zum Schaden seiner Passagiere, sondern im Gegenteil um den Geschäftsbetrieb für die Passagiere so angenehm wie möglich zu machen, auch wenn keine Online Systeme zur Verfügung stehen.

Insofern kann man durch dieses Beispiel weiter argumentieren, dass eine Umsetzung der EID wohl nicht für jeden Geschäftszweck sinnvoll ist und es berechtigterweise weiterhin Vornamen, Nachnamen und Geburtstage auf Billetten brauchen wird.
